Sources/SpeziOnboarding/OnboardingTitleView.swift

```diff
@@ -18,6 +18,7 @@ import SwiftUI
 public struct OnboardingTitleView: View {
     private let title: String
     private let subtitle: String?
+    private let paddingTop: CGFloat
```

Comment:
Looking at these changes, I think that we should remove the custom padding in this view completely. This speaks a bit agains the SwiftUI composability aspect, I would rather suggest that the views using the title view add an appropriate padding. We can add some padding between the title and the subtitle and maybe around the complete view (vertical) but should probably avoid the e.g. very specific 30pt padding.